app/models/building_operator.rb

```diff
@@ -63,6 +63,14 @@ def buildings
     buildings.to_a
   end
 
+  def categories
```

Comment:
this seems a bit repetitive between asset manager and building operator, maybe this function better belongs in Portfolio or BuildingTypes or somewhere where you can pass in the user_type and get specific results instead? Or I guess I just don't understand what asset_manager.categories should mean

Reply:
since asset_managers could be "building operators" in that jp morgan business model, we need them to be able to store their categories (which are decided off of the buildings that they have)